.github/workflows/pilotWrapper.yml

```
@@ -0,0 +1,26 @@
+name: pilot wrapper
+
+on: [push, pull_request]
+
+jobs:
+  PilotWrapper:
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: False
+      matrix:
+        python:
+          - 2.6.9
+          - 2.7.5
+          - 2.7.13
+          - 3.6.8
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: prepare environment
+      run: conda create -c conda-forge -n python_${{ matrix.python }} python=${{ matrix.python }}
```

By default conda adds `pip` as a dependency of `python` but that isn't what you want when using the old Pythons. This works around it:
```suggestion
      # Disable add_pip_as_python_dependency to prevent conflicts when testing with Python 2.6
      run: |
        conda config --set add_pip_as_python_dependency false
        conda create -c conda-forge -n python_${{ matrix.python }} python=${{ matrix.python }}
```

---

And I've edited the comment to add the `free` channel so you can find `2.7.5`. See here: https://docs.conda.io/projects/conda/en/latest/user-guide/configuration/free-channel.html